src/main/java/mat/client/shared/CQLLibraryResultTable.java

```diff
@@ -117,7 +117,7 @@ public void onBrowserEvent(Cell.Context context, Element elem, CQLLibraryDataSet
                 if (object.getLastClick() < System.currentTimeMillis() - DELAY_TIME) {
                     object.setSelected(!object.isSelected());
                     selectionModel.setSelected(object, object.isSelected());
-                } else {
+                } else if (object.isEditable()) {
```

Comment:
as I remember some of read-only items should be clickable/vieweable. How is this compatible with isEditable() ?

Reply:
Double Click works similar to edit button in grid toolbar.

Reply:
yeah, I just wanted to dobule check that it's not breaking anything
1. is the item editable to allow double click? What about isViewable read-only times?
2. is the change to isEditable affecting how toolbar's 'Edit' button enable/disable logic works?

Reply:
This is the requirement. "When the flag is off, the double click of a measure grid row does not launch the edit / view of a measure if the measure is a fhir measure but does for a QDM measure."

It doesn't change any existing logic

Reply:
The requirements might be inconsistent.
Just make sure that 'View' functionality is not broken after that (you can see how it works on the measures you don't own in matdev).

Reply:
ooh ok, I will check it

Reply:
View functionality is not broken from Grid button, but double of a measure which is viewable cannot be viewed. Since according to the requirement, edit / view of measure is disabled when double clicked.

Reply:
Are you sure?

There was a bug fixed recently to make it work https://jira.cms.gov/browse/MAT-422

Reply:
That implies that, double click is not disabled in any case ?

Reply:
that's my understanding. you can clarify with Kathy

Reply:
No Kathy says that, no matter what if the measure is FHIR, users cannot view or edit it.
From Kathy "If the fhiredit flag is off, we can't edit or view the fhir measure"

Reply:
ok, great. thanks for the confirmation